lib/transaction.js

```diff
@@ -85,7 +85,7 @@ Transaction.begin = function(connector, options, cb) {
       return cb(err);
     }
     var tx = connection;
-    if (!(connection instanceof Transaction)) {
+    if (!(connection instanceof Transaction) && !(typeof connection.txId === 'string')) {
```

Comment:
I would change the entirety of this check to assert against some common properties of the connection that indicate it's a Transaction. Given that we're seeing multiple copies of the loopback-connector in the dependency tree of some installations, using `instanceof` to do your check is potentially fraught with dangers.